utils/land-patterns.stanza

```diff
@@ -151,6 +151,34 @@ public defn rect-pth-pad (drill-r:Double, pad-w:Double, pad-h:Double) :
 public defn rect-pth-pad (drill-w:Double, drill-h:Double, pad-w:Double, pad-h:Double) :
   rect-pth-pad(C, drill-w, drill-h, pad-w, pad-h)
 
+pcb-pad roundrect-pth-pad (anchor:Anchor, drill-w:Double, drill-h:Double, pad-w:Double, pad-h:Double, pad-r:Double) :
+  name = to-string("%_ RoundedRectangle PTH Pad (drill = (%_, %_), pad = (%_, %_, %_))" % [
+           anchor, drill-w, drill-h, pad-w, pad-h, pad-r])
+  type = TH
+  if drill-w != drill-h :
+    fatal("roundrect-pth-pad(): RoundedRectangle holes are not supported")
+  
+  shape = RoundedRectangle(pad-w, pad-h, pad-r)
+  layer(Cutout()) = Circle(anchor, drill-w)
+  layer(SolderMask(Top)) = RoundedRectangle(pad-w, pad-h, pad-r)
```

Comment:
I followed what was done for ovals. Do we want the same layers to be filled?

Reply:
I think this is OK for now but we will likely change most of this when improving landpattern generation.